package.json

```diff
@@ -82,7 +82,11 @@
       "node tools/fetch-docs-links.js",
       "git add docs/src/_data/further_reading_links.json"
     ],
-    "docs/**/*.svg": "trunk check --fix --filter=svgo"
+    "docs/**/*.svg": "trunk check --fix --filter=svgo",
+    "{lib/rules/*.js,docs/src/_data/rule_versions.json}": [
+      "node tools/update-rule-type-headers.js",
+      "git add lib/types/rules/*.ts"
+    ]
```

Comment:
`docs/src/_data/rule_versions.json` is updated only during releases, so I think we can run this only for `lib/rules/*.js`, i.e., merge these commands into:

https://github.com/eslint/eslint/blob/b04b84bc17d4aaaea1326cb08196593624db02a2/package.json#L76-L79

Reply:
Done in e54e464745942848d333572ba36ad9c6b02134c0.